README.md

```diff
@@ -1961,6 +1993,16 @@ IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2020. [YouTube](https
 - <a name="Deng21tip"></a>Deng, Y., Chen, H., Chen, H. and Li, Y.,  
 *[Learning From Images: A Distillation Learning Framework for Event Cameras](https://ieeexplore.ieee.org/abstract/document/9426390)*,  
 IEEE Trans. Image Process. (TIP), 30:4919-4931, 2021. [Dataset](https://github.com/SuppDoc/EventKD)
+- <a name="schaefer22cvpr"></a>S. Schaefer*, D. Gehrig*, D. Scaramuzza,  
+*[AEGNN: Asynchronous Event-based Graph Neural Networks](https://arxiv.org/abs/2203.17149)*,  
+IEEE Conference of Computer Vision and Pattern Recognition (CVPR), 2022. [PDF](https://rpg.ifi.uzh.ch/docs/CVPR22_Schaefer.pdf), [YouTube](https://youtu.be/opbFE6OsAeA), [code](https://github.com/uzh-rpg/aegnn/).  
+- <a name="gehrig22arxiv_graph"></a>Daniel Gehrig, Davide Scaramuzza,  
+*[Pushing the Limits of Asynchronous Graph-based Object Detection with Event Cameras](https://arxiv.org/abs/2211.12324)*,  
```

Comment:
In arxiv-only entries, the title should have the link to the pdf directly. 
There is no need to have two links: one in the (long) title, pointing to /abs/ and one after the conference name, with a short "PDF" word pointing to /pdf/